services/IoT/applications/libraries/java.md

```diff
@@ -311,22 +311,22 @@ The following code sample provides an example implementation of the Status callb
   }
 ```
 
-When the status callback is added to the ApplicationClient, the processDeviceStatus() method is invoked whenever any device is connected or disconnected from the {{site.data.keyword.iot_short_notm}} that matches the criteria. The following code sample shows how you can add the status call back instance into ApplicationClient:
+When the status callback is added to the ApplicationClient, the processDeviceStatus() method is invoked whenever any device is connected or disconnected from the {{site.data.keyword.iot_short_notm}} that matches the criteria. The following code sample shows how you can add the status callback instance to the ApplicationClient:
 
 ```
 
     myClient.connect()
     myClient.setStatusCallback(new MyStatusCallback());
     myClient.subscribeToDeviceStatus();
 ```
-As with device status, the application can also subscribe to any other application connect or disconnect status. The following code snippet shows how to subscribe to the application status in the organization:
+Line the device status, the application can also subscribe to any other application connect or disconnect status. The following code snippet shows how to subscribe to the application status in the organization:
 
 ```
     myClient.connect()
     myClient.setEventCallback(new MyEventCallback());
     myClient.subscribeToApplicationStatus();
 ```
-Overloaded method is available to control the status subscription to a particular application. The processApplicationStatus() method is called whenever any application is connected or disconnected from the {{site.data.keyword.iot_short_notm}} that matches the criteria.
+The overloaded method is available to control the status subscription to a particular application. The processApplicationStatus() method is called whenever any application is connected or disconnected from the {{site.data.keyword.iot_short_notm}} that matches the criteria.
```

Comment:
"that matches the criteria" is dangling at the end and it's hard to see what it belongs to. Clarify.